### ISSUE TITLE
[Feature Request]: Add efficient upload to Dropbox

### ISSUE THREAD

**Issue by sboesen** [sboesen/remotely-sync#74]
### What feature are you suggesting?

Large vaults trigger mass 429s and eventually time out. Fix this by batch uploading

### What remote cloud services are you using, or suggesting adding the feature to?

Dropbox

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by kadisonm** [comment]
Thanks, I also got this issue.

**Comment by sboesen** [comment]
Yeah unfortunately means somewhat more complexity in how we upload stuff. I think reducing concurrency is a workaround until then
